### DISCUSSION TITLE
when deploying project in versel  un able to deploy  it

### DISCUSSION THREAD

**Discussion by Avinashj14** [vercel/community#5761] [Help] [answered]
### Summary

As i was trying to deploy my proect through versel i got Such error  below  :-   i have uploded my code in    it you can try uploding it   
https://github.com/Avinashj14/FirebaseRedux-Todo

please hepl me with that

npm ERR! code ERESOLVE
--
14:38:44.453 | npm ERR! ERESOLVE could not resolve
14:38:44.454 | npm ERR!
14:38:44.454 | npm ERR! While resolving: redux-devtools-extension@2.13.9
14:38:44.454 | npm ERR! Found: redux@5.0.1
14:38:44.454 | npm ERR! node_modules/redux
14:38:44.454 | npm ERR!   redux@"^5.0.1" from the root project
14:38:44.455 | npm ERR!   peerOptional redux@"^5.0.0" from react-redux@9.1.0
14:38:44.455 | npm ERR!   node_modules/react-redux
14:38:44.455 | npm ERR!     react-redux@"^9.1.0" from the root project
14:38:44.455 | npm ERR!   1 more (redux-thunk)
14:38:44.455 | npm ERR!
14:38:44.456 | npm ERR! Could not resolve dependency:
14:38:44.456 | npm ERR! peer redux@"^3.1.0 \|\| ^4.0.0" from redux-devtools-extension@2.13.9
14:38:44.456 | npm ERR! node_modules/redux-devtools-extension
14:38:44.456 | npm ERR!   redux-devtools-extension@"^2.13.9" from the root project
14:38:44.456 | npm ERR!
14:38:44.456 | npm ERR! Conflicting peer dependency: redux@4.2.1
14:38:44.456 | npm ERR! node_modules/redux
14:38:44.457 | npm ERR!   peer redux@"^3.1.0 \|\| ^4.0.0" from redux-devtools-extension@2.13.9
14:38:44.457 | npm ERR!   node_modules/redux-devtools-extension

### Steps to Reproduce

my code in    it you can try uploding it   
https://github.com/Avinashj14/FirebaseRedux-Todo

all project where i have apply redux gives that error 
![asdasd](https://github.com/vercel/community/assets/89546465/3b832952-b48f-4ac3-83ae-51081fe5c934)

**Comment by amyegan** [comment, marked as answer]
Hey @Avinashj14. This is a problem with project dependencies. I get the same error if I run `npm install` on a local copy of the repo you shared. You'll need to take a look at the specific dependency conflicts reported and make adjustments to get it working.
